samples/js-valueentity-shopping-cart/proto/shoppingcart_api.proto

```diff
@@ -14,14 +14,21 @@
 
 // This is the public API offered by the shopping cart Value Entity.
 
+// tag::proto_syntax[]
 syntax = "proto3";
+// end::proto_syntax[]
 
+// tag::proto_package[]
 package com.example.shoppingcart;
+// end::proto_package[]
 
-import "google/protobuf/empty.proto";
+// tag::proto_imports[]
 import "akkaserverless/annotations.proto";
 import "google/api/annotations.proto";
+import "google/protobuf/empty.proto";
```

Comment:
When changing the Sunglasses UI, I ran into an issue where the ordering of imports matter. The akkaserverless import needed to come after the google annotations one. 

I think this was more a problem of grpc-web, because this ordering is known to work here. But perhaps good to change to avoid issues. 

```suggestion
import "google/api/annotations.proto";
import "google/protobuf/empty.proto";
import "akkaserverless/annotations.proto";
```

Reply:
Wow.